### PR TITLE
peer code review

### PR DESCRIPTION
**Pull request by morganlloyd**
- wondering if i should add more notes throughout my components
- MVP: adding and deleting affirmations 
- SG: light / dark theme switch
- another SG: I wanted to create a like counter for each of the affirmation items - but was unable to get the state to maintain on browser refresh so I scraped that idea for now. Any ideas on logic for that?

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+
+
 const Form = () => {
+
+    const [affirmationInput, setAffirmationInput] = useState("");
+
+    const handleChange = (e) => {
+        setAffirmationInput(e.target.value)
+
+        // console.log(e.target.value);
```
Comment (jasmine-lmj) [inline src/components/Form.js]: Might want to remove all the console.logs

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,242 @@
...
+}
+
+.delete:hover,
+.delete:focus {
+  color: #d7543d;
+}
+
+
+
+/* LIGHT/DARK THEME */
```
Comment (jhandrahan) [inline src/index.css:180]: Absolutely love the dark and light theme! Nice Touch! 😊

File: src/components/LikeCounter.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const LikeCounter = () => {
```
Comment (jasmine-lmj) [inline src/components/LikeCounter.js]: Not sure if you need this component.

**Review by jhandrahan — APPROVED** [review]
Honestly your code is beautiful. Its clean and organized throughout with lots of comments to explain whats going on. The app on chrome desktop and my iphone work great. It's responsive and runs smoothly. Great Job! 😁
The only thing i Noticed on my phone is the Title alignment is kinds weird on mobile the word "Believe" is too close to the right edge, looks asymmetrical

File: src/index.css [hunk 4]
```diff
@@ -1,13 +1,242 @@
...
+
+.titleStyles {
+  position: relative;
+  right: -50px;
+
+}
+
+
+
+/* FORM */
```
Comment (jasmine-lmj) [inline src/index.css:82]: Love the pseudo codes!

**Pull request merged**